packages/sdk/src/dataProtector/IExecDataProtector.ts

```diff
@@ -21,10 +21,11 @@ import { addToCollection } from './sharing/addToCollection.js';
 import { createCollection } from './sharing/createCollection.js';
 import { getSubscribers } from './sharing/getSubscribers.js';
 import { setProtectedDataToSubscription } from './sharing/setProtectedDataToSubscription.js';
-import { setSubscriptionOptions } from './sharing/setSubscriptionOptions.js';
+import { setSubscriptionParams } from './sharing/setSubscriptionParams.js';
 import { saveForSharingContract } from './sharing/smartContract/getSharingContract.js';
 import { getCollectionsByOwner } from './sharing/subgraph/getCollectionsByOwner.js';
 import { getCreators } from './sharing/subgraph/getCreators.js';
+import { getRenters } from './sharing/subgraph/getRenters.js';
```

Comment:
Just to confirm, are we ok to expose methods directly coming from a subgraph query? (I would say yes)

Reply:
Yeah  think